generation/um/sysinfoapi/generate.rsp

```diff
@@ -0,0 +1,42 @@
+--additional
+-m32
+--config
+multi-file
+--file
+um-sysinfoapi.h
+--headerFile
+header.txt
+--include-directory
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/shared
+--include-directory
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/um
+--libraryPath
+kernel32
+--methodClassName
+Kernel32
+--namespace
+TerraFX.Interop
+--output
+../../../sources/Interop/Kernel32/um/sysinfoapi
+--remap
+__AnonymousField_sysinfoapi_L48_C5=Anonymous
+__AnonymousField_sysinfoapi_L50_C9=Anonymous
+__AnonymousRecord_sysinfoapi_L48_C5=_Anonymous_e__Union1
+__AnonymousRecord_sysinfoapi_L50_C9=_Anonymous_e__Union2
+_COMPUTER_NAME_FORMAT=COMPUTER_NAME_FORMAT
+_FILETIME=FILETIME
+_LOGICAL_PROCESSOR_RELATIONSHIP=LOGICAL_PROCESSOR_RELATIONSHIP
+_MEMORYSTATUSEX=MEMORYSTATUSEX
+_OSVERSIONINFOA=OSVERSIONINFOA
+_OSVERSIONINFOW=OSVERSIONINFOW
+_SYSTEM_INFO=SYSTEM_INFO
+_SYSTEM_LOGICAL_PROCESSOR_INFORMATION=SYSTEM_LOGICAL_PROCESSOR_INFORMATION
+_SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX=SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX
+_SYSTEM_PROCESSOR_CYCLE_TIME_INFORMATION=SYSTEM_PROCESSOR_CYCLE_TIME_INFORMATION
+_SYSTEMTIME=SYSTEMTIME
+DWORD_PTR=uint*
```

Comment:
Was this one not handled automatically?

Reply:
Correct. I just generated bindings again without `DWORD_PTR` and it generated `uint`.

Reply:
Looks like that's because it is `typedef ULONG_PTR DWORD_PTR` and so therefore should be `UIntPtr`

Reply:
As discussed, I changed the mapping to `DWORD_PTR=UIntPtr`.